src/app/companies/companies.component.html

```diff
@@ -1 +1,21 @@
+<div fxLayout="row" fxLayoutAlign="space-evenly center" fxLayoutGap="10px" fxLayoutWrap class="companies-container">
+  <mat-card *ngFor="let company of companies | async" fxFlex.xs="100" fxFlex.sm="45" fxFlex.md="30" fxFlex.lg="23">
+    <mat-card-header>
+      <div mat-card-avatar>
+        <img [src]="company.logo" class="company-logo">
+      </div>
+      <mat-card-title>{{company.name}}</mat-card-title>
+      <mat-card-subtitle>
+        {{company.location}}
+        <button class="website-button" mat-icon-button aria-label="website" (click)="openWindow(company.website)">
```

Comment:
For better accessibility this should be an a - tag with a proper href

Reply:
Ah, my bad. I was updating an existing PR and missed this. Will push up the changes.